modules/ding_webtrekk/ding_webtrekk.js

```diff
@@ -28,6 +28,10 @@
   };
 
   var pushEvent = function(event, eventData) {
+    if (Drupal.settings.dingWebtrekk.debugMode) {
+      console.log('Ding webtrekk - Pushing "' + event +  '"-event with data:');
+      console.log(JSON.stringify(eventData, null, 2))
```

Comment:
[`console.log`](https://developer.mozilla.org/en-US/docs/Web/API/console#Outputting_text_to_the_console) accepts all sorts of arguments. Is there a reason this is not just:

```js
console.log('Ding webtrekk - Pushing ', event, "-event with data:', eventData);
```

Reply:
If I remember correctly the JSON.stringify produced cleaner more readable output. I'l make some screenshots..

Reply:
**JSON.stringify**
![3573-debug-json-stringify](https://user-images.githubusercontent.com/5011234/63092892-89700c00-bf63-11e9-9ee9-246b119df881.PNG)

**Console.log argument**
![3573-debug-console-log-argument](https://user-images.githubusercontent.com/5011234/63092907-97259180-bf63-11e9-9e38-6c7c50324957.PNG)

Reply:
I think JSON.stringify looks better in this case. The collapsible object structure is a bit overkill, since we're only interesting in seeing the data in the data layer object.

Reply:
Fixed. Using console arguments and JSON.stringify

Reply:
:+1: